pkgs/desktops/gnome-3/extensions/default.nix

```diff
@@ -0,0 +1,25 @@
+{pkgs, ...}: let
+  # Index of all scraped extensions (with supported versions)
+  extensionsIndex = builtins.fromJSON (builtins.readFile ./extensions.json);
```

Comment:
Use `lib.importJSON` instead (which should really have been called `loadJSON` but well...)